--- origsrc/libbluray-0.2.2/src/Makefile.am	2012-03-15 11:02:31.000000000 -0500
+++ src/libbluray-0.2.2/src/Makefile.am	2012-05-29 13:13:29.006516200 -0500
@@ -88,7 +88,7 @@ libbluray_la_SOURCES=libbluray/bluray.h
 	util/bits.h \
 	util/logging.h
 libbluray_la_HEADERS= libbluray/bluray.h file/filesystem.h util/log_control.h libbluray/keys.h libbluray/decoders/overlay.h libbluray/bdnav/meta_data.h libbluray/bdnav/clpi_data.h libbluray/bluray-version.h
-libbluray_la_LDFLAGS= -version-info $(LIB_VERSION_INFO) $(DLOPEN_LDFLAGS)
+libbluray_la_LDFLAGS= -version-info $(LIB_VERSION_INFO) -no-undefined $(DLOPEN_LDFLAGS)
 libbluray_la_LIBADD= $(LIBXML2_LIBS)
 
 if USING_BDJAVA
